ArduCopter/mode_auto.cpp

```diff
@@ -857,23 +857,23 @@ uint32_t ModeAuto::wp_distance() const
 {
     switch (_mode) {
     case SubMode::CIRCLE:
-        return copter.circle_nav->get_distance_to_target();
+        return copter.circle_nav->get_distance_to_target_cm();
     case SubMode::WP:
     case SubMode::CIRCLE_MOVE_TO_EDGE:
     default:
-        return wp_nav->get_wp_distance_to_destination();
+        return wp_nav->get_wp_distance_to_destination_cm();
     }
 }
 
 int32_t ModeAuto::wp_bearing() const
 {
     switch (_mode) {
     case SubMode::CIRCLE:
-        return copter.circle_nav->get_bearing_to_target();
+        return copter.circle_nav->get_bearing_to_target_cd();
     case SubMode::WP:
     case SubMode::CIRCLE_MOVE_TO_EDGE:
     default:
-        return wp_nav->get_wp_bearing_to_destination();
+        return wp_nav->get_wp_bearing_to_destination_cdeg();
```

Comment:
We see to have both `cd` and `cdeg` - should we choose one?

Reply:
we decided that it should be "cd"